packages/dart/lib/src/objects/parse_error.dart

```diff
@@ -13,58 +13,279 @@ class ParseError {
+
+  /// Error code indicating that a field had an invalid value.
+  static const int invalidValue = 162;
+
+  /// Error code indicating that the username is missing or empty.
+  static const int usernameMissing = 200;
+
+  /// Error code indicating that the password is missing or empty.
+  static const int passwordMissing = 201;
+
+  /// Error code indicating that the username has already been taken.
+  static const int usernameTaken = 202;
+
+  /// Error code indicating that the email has already been taken.
+  static const int emailTaken = 203;
+
+  /// Error code indicating that the email is missing, but must be specified.
+  static const int emailMissing = 204;
+
+  /// Error code indicating that a user with the specified email was not found.
+  static const int emailNotFound = 205;
+
+  /// Error code indicating that a user object without a valid session could
+  /// not be altered.
+  static const int sessionMissing = 206;
+
+  /// Error code indicating that a user can only be created through signup.
+  static const int mustCreateUserThroughSignup = 207;
+
+  /// Error code indicating that an an account being linked is already linked
+  /// to another user.
+  static const int accountAlreadyLinked = 208;
+
+  /// Error code indicating that the current session token is invalid.
+  static const int invalidSessionToken = 209;
+
+  /// Error code indicating an error enabling or verifying MFA
+  static const int mfaError = 210;
+
+  /// Error code indicating that a valid MFA token must be provided
+  static const int mfaTokenRequired = 211;
+
+  /// Error code indicating that a user cannot be linked to an account because
+  /// that account's id could not be found.
+  static const int linkedIdMissing = 250;
+
+  /// Error code indicating that a user with a linked (e.g. Facebook) account
+  /// has an invalid session.
+  static const int invalidLinkedSession = 251;
+
+  /// Error code indicating that a service being linked (e.g. Facebook or
+  /// Twitter) is unsupported.
+  static const int unsupportedService = 252;
+
+  /// Error code indicating an invalid operation occured on schema
+  static const int invalidSchemaOperation = 255;
+
+  /// Error code indicating that there were multiple errors. Aggregate errors
+  /// have an "errors" property, which is an array of error objects with more
+  /// detail about each error that occurred.
+  static const int aggregateError = 600;
+
+  /// Error code indicating the client was unable to read an input file.
+  static const int fileReadError = 601;
+
+  /// Error code indicating a real error code is unavailable because
+  /// we had to use an XDomainRequest object to allow CORS requests in
+  /// Internet Explorer, which strips the body from HTTP responses that have
+  /// a non-2XX status code.
+  static const int xDomainRequest = 602;
+
   static const Map<int, String> _exceptions = {
-    -1: 'UnknownError',
-
-    // SDK errors / Errors
-    1: 'No Results',
```

Comment:
It seems that these error codes (which you already removed) were actually some Flutter SDK-custom codes (1, 2) and a HTTP response code (400).

Have these 3 codes been used anywhere? If so, this is a breaking change, at least for code 1 which was described as "No Results" but is now "internalServerError".

Reply:
I don't think it makes sense to return them, the 400 code is incorrect anyway, because it mixes a HTTP response code into the Parse Error codes. If you are 100% sure they weren't used anywhere we can just remove them. Did you also do a simple text search to be sure?

Reply:
yes 
In addition, I saw error 400 in several issues, none of which were related to code 400 https://github.com/parse-community/Parse-SDK-Flutter/issues/487#issue-721516490  ,https://github.com/parse-community/Parse-SDK-Flutter/issues/487#issue-721516490 and https://github.com/parse-community/Parse-SDK-Flutter/issues/481#issue-719863736

contributors parse_error.dart are codes 1, 2 and 400 important?
@nstrelow 
@fischerscode

Reply:
@mtrezza 
I suggest not to add codes unless there is a problem because I don't see any use of these codes in Parse Flutter!

Reply:
Sounds good